### PR TITLE
Fixing Paths and other documentation issues

### PR DESCRIPTION
**Pull request by ShorthillsAI**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Added new unit/integration tests
- [ ] Added new notebook (that tests end-to-end)
- [ ] I stared at the code and made sure it makes sense

# Suggested Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added Google Colab support for the newly added notebooks.
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I ran `make format; make lint` to appease the lint gods

### CODE REVIEW

File: docs/use_cases/q_and_a.md [hunk 1]
```diff
@@ -17,10 +17,10 @@ Q&A has all sorts of sub-types, such as:
 
 - **Over documents**: LlamaIndex can pull in unstructured text, PDFs, Notion and Slack documents and more and index the data within them.
   - [Example of search over documents](combine-multiple-sources)
-  - [Building a multi-document agent over the LlamaIndex docs](/examples/agent/multi_document_agents-v1.ipynb)
+  - [Building a multi-document agent over the LlamaIndex docs](/docs/examples/agent/multi_document_agents-v1.ipynb)
```
Comment (logan-markewich) [inline docs/use_cases/q_and_a.md:20]: I'm actually convinced that adding this `docs` prefix is actually breaking the docs 😅 

Let's see a few examples where it's already used
- The links here, which don't work: https://docs.llamaindex.ai/en/stable/changes/deprecated_terms.html
- The links here, which don't work: https://docs.llamaindex.ai/en/stable/use_cases/extraction.html
- The link at the bottom, which doesn't work: https://docs.llamaindex.ai/en/stable/use_cases/agents.html

Going to investigate the cause 🤔 It seems an extreme amount of links are broken

**Comment by ShorthillsAI** [comment]
@logan-markewich, Just a reminder, Any update on this, what steps needs to be taken now?

**Comment by ShorthillsAI** [comment]
@logan-markewich , Any update?

**Comment by logan-markewich** [comment]
@ShorthillsAI let's revert the changes that use `/docs` prefix so that we can merge, and figure out a proper fix for those links in another pr

**Comment by ShorthillsAI** [comment]
[like]  Aashish Saini reacted to your message:
________________________________

**Review by logan-markewich — APPROVED** [review]

**Comment by logan-markewich** [comment]
@ShorthillsAI cool, thanks!

**Pull request merged**